log4j-core/src/main/java/org/apache/logging/log4j/core/util/UuidUtil.java

```diff
@@ -65,6 +65,20 @@
 
     static {
         byte[] mac = NetUtils.getMacAddress();
+        LEAST = initialize(mac);
+    }
```

Comment:
This looks much better, thanks! Any chance we can replace the static block entirely?

```java
private static final long LEAST = initialize(NetUtils.getMacAddress());
```

Reply:
Done.